.gitignore

```diff
@@ -1,3 +1,12 @@
 /build
 /extension/version_auto.h
 /breakpad
+/package
+/test
+/test-crash-dump-generation
+/accelerator.ext
+
+.ambuild
+.vscode
+
+pdblog.txt
```

Comment:
FYI, you shouldn't be building in the checkout directory - create a directory for the build (either outside the checkout directory, or `build/` which is already ignored) and run `./configure.py` from there.